app/models/concerns/resource_utilizations/validation.rb

```diff
@@ -7,5 +7,7 @@ module ResourceUtilizations::Validation
       greater_than_or_equal_to: 0,
       less_than_or_equal_to: 9_999_999_999_999.99
     }, allow_nil: true, allow_blank: true
+    validates :resource_id, uniqueness: { scope: [:resource_consumer_type,
+              :resource_consumer_id, :resource_type] }
```

Comment:
No decía así, sino así:
```suggestion
    validates :resource_id, uniqueness: {
      scope: [:resource_consumer_type, :resource_consumer_id, :resource_type]
    }
```
No llego a contar por acá, pero lo dejamos y cualquier cosa después lo modificamos.